### PR TITLE
Delete files from blob storage

### PR DESCRIPTION
**Pull request by adamdougal**
Fixes https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/issues/448

### CODE REVIEW

File: code/backend/pages/03_Delete_Data.py [hunk 1]
```diff
@@ -78,7 +80,23 @@ def delete_files(files):
     st.success("Deleted files: " + str(files_to_delete))
 
 
+def get_filename(file: str) -> str:
```
Comment (ross-p-smith) [inline code/backend/pages/03_Delete_Data.py:83]: Just parsing the code in my head - will this work with @komalg1 IV work where the blob could be stored in a URI format?

Comment (adamdougal) [reply code/backend/pages/03_Delete_Data.py:83]: Ah good point. Let me check on that!

File: code/backend/pages/03_Delete_Data.py [hunk 2]
```diff
@@ -78,7 +80,23 @@ def delete_files(files):
...
+    /container_name/file_name
+    to return the file name.
+
+    Args:
+        file (str): The file path of the file to delete.
+
+    Returns:
+        filename (str): The filename part of the file path
+    """
+    return file.split("/")[2]
```
Comment (superhindupur) [inline code/backend/pages/03_Delete_Data.py:95]: What is the behavior if, for some reason, the split or the [2] doesn't work? Will the code throw an error and never get to the search_client.delete_documents()?

**Comment by adamdougal** [comment]
I think I'm going to hold off on this for now. Quite a few moving parts and more involved then anticipated for a quick win!

**Pull request closed without merging**